timesketch/ui/templates/home/home.html

```diff
@@ -0,0 +1,73 @@
+<!--
+Copyright 2015 Google Inc. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+{% extends "base.html" %}
+
+{% block sidenav %}
+<div id="navigation">
+  <div class="leftnav-title">
+    <form action="{{ url_for('home_views.home') }}" method="post">
+      {{ form.name }}
```

Comment:
Agree, and yes there is a WIP to move over to pure client side templates, but there are some things that needs to be done before this can happen. Moving this to a future PR.